### PR TITLE
Update login-loop.md

### PR DESCRIPTION
**Pull request by CodeStardust**
Add troubleshooting step for users using an AMD graphics driver. 

Found this solution after doing some research and it worked for me.

### CODE REVIEW

File: _articles/login-loop.md [hunk 1]
```diff
@@ -117,6 +118,14 @@ sudo apt install system76-driver-nvidia
 
 After the installation has completed, type `sudo systemctl reboot` and try logging in again.
 
+### Blacklist Radeon driver
+If you are using an AMD graphics card, the radeon driver might be causing issues as it does not support newer versions of Ubuntu.  The amdgpu driver that comes with the kernel does however. To blacklist the radeon driver and ensure amdgpu is loaded:
+
+* Open `/etc/modprobe.d/blacklist.conf` for editing.
+* Add `blacklist radeon` to the file
+* Run `sudo update-initramfs -u`
```
Comment (storyaddict) [inline _articles/login-loop.md:126]: Run 'sudo update-initramfs -c -k all'

**Comment by CodeStardust** [comment]
thx! 😄

**Comment by ahoneybun** [comment]
@CodeStardust it looks like there are some conflicts in this article on your branch can you use this article to fix this issue then we can merge it? Thank you.

https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line

**Comment by CodeStardust** [comment]
Yes @ahoneybun  I will do so after work tonight!

**Pull request merged**